MAUI/MauiAppBasic/MSALClient/AppConstants.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;

```

Comment:
>using [](http://example.com/codeflow?start=0&length=5)

In PCAWrapper.cs, add the following in the PCA initialization

WithAuthority(()
WithReplyUrl()

Reply:
Already included.

The **redirect uri** is platform specific and handled in the code already.